src/Features/CSharp/Completion/CompletionProviders/SuggestionModeCompletionProvider.cs

```diff
@@ -66,7 +66,7 @@ public TextChange GetTextChange(CompletionItem selectedItem, char? ch = null, st
             {
                 var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
 
-                if (triggerInfo.IsDebugger && triggerInfo.IsImmediateWindow)
+                if (triggerInfo.IsDebugger)
                 {
                     // Aggressive Intellisense in the debugger: always show the builder 
```

Comment:
Interestingly, VB didn't have the "&& triggerInfo.IsImmediateWindow" condition (so nothing to change there).

Per unit tests, I'm waiting for the buddy build to find out... :wink: